django_freeradius/base/models.py

```diff
@@ -819,3 +820,36 @@ def _remove_files(self):
         path = getattr(self, strategy_filemap.get(self.strategy)).path
         if os.path.isfile(path):
             os.remove(path)
+
+
+class AbstractRadiusToken(AutoUsernameMixin, models.Model):
+    key = models.CharField(_("Key"), max_length=40, primary_key=True)
+
+    username = models.CharField(verbose_name=_('username'),
+                                max_length=64,
+                                db_index=True,
+                                blank=True)
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE,
+                             blank=True,
+                             null=True)
+
+    created = models.DateTimeField(_("Created"), auto_now_add=True)
+
+    class Meta:
+        db_table = 'radtoken'
+        verbose_name = _("radius token")
+        verbose_name_plural = _("radius tokens")
```

Comment:
aren't we using single quotes in the file? Please keep the style of quotes consistent